.travis.yml

```diff
@@ -28,21 +29,19 @@ matrix:
 
     # Do one build with sphinx-astropy as one of the tests bypasses the auto-
     # installation but we want to make sure that test runs for coverage.
-    - os: linux
-      env: PYTHON_VERSION=3.6 PIP_DEPENDENCIES='codecov'
```

Comment:
Not sure what this is meant to test. Seems a little redundant, but I can add it back if you want.